package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "clock",
-  "version": "1.0.0",
+  "name": "@first-lego-league/clock",
+  "version": "0.0.0",
```

Comment:
backnumbering versions is a bad idea. What is the reason for this? It should go forward only

Reply:
I have backtrack it. because i didn't want to upload a beta version of the code as version 1. I believe it's ok since the I was the first to upload it to npm under this name.

Reply:
I would propose the following

- change the version to 1.0.x
- name is ok like this
- create a PR, I will merge that to gh-pages

And then

- create a beta branch, versioned 1.1.0-beta.1
- create a PR, I will merge that into a beta branch
- publish that beta on npm if you wish

That way, the versions are increasing always. Note that 1.1.0-beta.1 is a lower version than 1.1.0 automatically

Reply:
The rationale is that I rather not publish beta versions anyway (for such a small project), but I 
can understand it may be beneficial to your progress. This way, the beta is still accessible (as a branch), but not in the danger of being used accidentally

Also, if you want, just publish the main branch (gh-pages) as version 1.0.x in npm. I don't really see the need for having a beta there in this case.